lib/gitter-adapter.js

```diff
@@ -193,7 +193,7 @@ Adapter.prototype.joinRoom = function(channel) {
     var channelType = '*'; // TODO set right type -> * private, @ secret, = public
 
     var usernames = users.map(function(u) { return u.username; });
-    usernames.push('gitter'); // Fake gitter user for Events
+    usernames.push( config.irc.user ); // Fake gitter user for Events
```

Comment:
You're missing a `require('./config')` on this file? Also check line number 72 to use `config.gitter` endpoint there.